storage/interface.go

```diff
@@ -59,3 +62,33 @@ type database interface {
 	getClass() Class
 	close()
 }
+
+var ErrNotFound = errors.New("not found")
+
+// SessionDatabase is an in memory database that holds session data on a KV basis.
```

Comment:
whether it's in-memory depends on the actual implementation, the interface isn't bound to that

Reply:
It can never be persistent. So that leaves it as in-memory? I'll add `(clustered)` to it.